types/fulu/block_contents.yaml

```diff
@@ -0,0 +1,26 @@
+Fulu:
+  BlockContents:
+    type: object
+    description: "The required object for block production according to the Fulu CL spec."
+    required: [block, kzg_proofs, blobs]
+    properties:
+      block:
+        $ref: "../electra/block.yaml#/Electra/BeaconBlock"
+      kzg_proofs:
+        description: "Cell proofs of the blobs as defined in EIP-7594"
+        $ref: "../deneb/block_contents.yaml#/Deneb/KZGProofs"
+      blobs:
+        $ref: "../deneb/block_contents.yaml#/Deneb/Blobs"
+
+  SignedBlockContents:
+    type: object
+    description: "The required signed components of block production according to the Fulu CL spec."
+    required: [signed_block, kzg_proofs, blobs]
+    properties:
+      signed_block:
+        $ref: "../electra/block.yaml#/Electra/SignedBeaconBlock"
+      kzg_proofs:
+        description: "Cell proofs of the blobs as defined in EIP-7594"
+        $ref: "../deneb/block_contents.yaml#/Deneb/KZGProofs"
```

Comment:
doesn't the list limit change in PeerDAS compared to Deneb?

Reply:
Great catch, updated